build/config/webos/pkg_config.gni

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2013 The Chromium Authors. All rights reserved.
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
what are the deltas from `build/config/linux/pkg_config.gni` ? can we reuse instead of using a copy?

Reply:
@andy31415 
As you suggested, I deleted below files and re-use the one from linux.
- build/config/webos/pkg_config.gni
- build/config/webos/pkg-config.py

Thank you for your review.